learning/features/networking/high_level_multiplayer.rst

```diff
@@ -215,39 +258,44 @@ In most games, each player will likely have its own scene. Remember that this is
 you need to instance **one scene for each player connected to it**. For a 4 player game, each peer needs to instance 4 player nodes.
 
 So, how to name such nodes? In Godot nodes need to have an unique name. It must also be relatively easy for a player to tell which
-nodes represent each player id.
+nodes represent each player ID.
 
 The solution is to simply name the *root nodes of the instanced player scenes as their network ID*. This way, they will be the same in 
 every peer and RPC will work great! Here is an example:
 
 ::
 
     remote func pre_configure_game():
+    	var selfPeerID = get_tree().get_network_unique_id()
+	
         # Load world
         var world = load(which_level).instance()
         get_node("/root").add_child(world)
 
         # Load my player
         var my_player = preload("res://player.tscn").instance()
-        my_player.set_name(str(get_tree().get_network_unique_id()))
-        my_player.set_network_mode(NETWORK_MODE_MASTER) # Will be explained later
+        my_player.set_name(str(selfPeerID))
+        my_player.set_network_master(selfPeerID) # Will be explained later
         get_node("/root/world/players").add_child(my_player)
 
         # Load other players
         for p in player_info:
             var player = preload("res://player.tscn").instance()
             player.set_name(str(p))
-            player.set_network_mode(NETWORK_MODE_SLAVE) # Will be explained later
             get_node("/root/world/players").add_child(player)
 
         # Tell server (remember, server is always ID=1) that this peer is done pre-configuring
-        rpc_id(1, "done_preconfiguring", get_tree().get_network_unique_id())
+        rpc_id(1, "done_preconfiguring", selfPeerID)
 	
-Synchronized game start
+
+.. note:: Depending on when you execute pre_configure_game(), you may need to change the any calls to add childs
+          to be deferred, as the SceneTree is locked while the scene is being created (e.g. when `_ready` is being called).
```

Comment:
Either "calls to ``add_child()``" or "calls to add children".  Use double backticks around code in RST.